doc/user-guide/climada_entity_MeasureSet.ipynb

```diff
@@ -553,103 +553,35 @@
+    "Measures defined in an excel file following the template provided in sheet `measures` of `climada_python/data/system/entity_template.xlsx` can be ingested directly using the method `from_excel()`. Measure sets can be written to file with the method `write_excel()`."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
    "metadata": {
     "ExecuteTime": {
      "end_time": "2021-03-05T12:22:41.336852Z",
      "start_time": "2021-03-05T12:22:41.266438Z"
     }
    },
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<climada.entity.measures.measure_set.MeasureSet at 0x1f30d913b80>"
-      ]
-     },
-     "execution_count": 1,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "from climada.entity.measures import MeasureSet\n",
-    "from climada.util import ENT_TEMPLATE_XLS\n",
-    "\n",
-    "# Fill DataFrame from Excel file\n",
-    "file_name = ENT_TEMPLATE_XLS  # provide absolute path of the excel file\n",
-    "meas_set = MeasureSet.from_excel(file_name)\n",
-    "meas_set"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Write measures\n",
-    "\n",
-    "Measures can be writen in Excel format using `write_excel()` method."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2021-03-05T11:44:20.460278Z",
-     "start_time": "2021-03-05T11:44:16.581Z"
-    }
-   },
    "outputs": [],
    "source": [
     "from climada.entity.measures import MeasureSet\n",
     "from climada.util import ENT_TEMPLATE_XLS\n",
     "\n",
     "# Fill DataFrame from Excel file\n",
     "file_name = ENT_TEMPLATE_XLS  # provide absolute path of the excel file\n",
-    "meas_set = MeasureSet.from_excel(file_name)\n",
-    "\n",
-    "# write file\n",
-    "meas_set.write_excel(\"results/tutorial_meas_set.xlsx\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "adb8d606",
-   "metadata": {},
-   "source": [
-    "Pickle can be used as well, but note that pickle has a [transient format](saving-with-pickle) and should be avoided when possible:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 9,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2021-03-05T11:44:20.462069Z",
-     "start_time": "2021-03-05T11:44:16.599Z"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "from climada.util.save import save\n",
-    "\n",
-    "# this generates a results folder in the current path and stores the output there\n",
-    "save(\"tutorial_meas_set.p\", meas_set)"
+    "meas_set = MeasureSet.from_excel(file_name)"
    ]
   }
  ],
  "metadata": {
   "hide_input": false,
   "kernelspec": {
-   "display_name": "Python 3.8.12 ('climada_core')",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
@chahank @spjuhel I think this is stuff that nbstripout would remove. Maybe we should reconsider?